src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,10 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+    private UserService userService = new UserService();
```

Comment:
Optional: Actually it is common practice to instantiate fields in constructor not in fields, such approach adds flexibility and, for example, allows you to replace a original class with its child class